engine-transactions/Cargo.toml

```diff
@@ -0,0 +1,24 @@
+[package]
+name = "aurora-engine-transactions"
+version = "1.0.0"
+authors = ["NEAR <hello@near.org>"]
```

Comment:
Do we not have our own author email?

Reply:
We should update that, its like that throughout our library. Update it here for now, then we can address the rest in a further PR.